version/src/lib.rs

```diff
@@ -120,7 +120,7 @@ pub struct UnsupportedVersion {
     /// Version of the content.
     pub version: u8,
     /// Raw content.
-    pub raw: RawVersioned,
+    pub raw: Box<RawVersioned>,
```

Comment:
what is the reason this is boxed now?

Reply:
Size of enum. Without it, the variant is huge compared to the other variant.

Reply:
as far as I can see that's not true:
```
pub enum RawVersioned {
    /// In JSON format.
    Json(String),
    /// In Parity Scale Codec format.
    ScaleBytes(Vec<u8>),
}
```

Reply:
it's just two extra bytes?